src/BuildScriptGenerator/SdkStorageVersionProviderBase.cs

```diff
@@ -97,9 +98,25 @@ protected string GetDefaultVersion(string platformName, string sdkStorageBaseUrl
             this.logger.LogDebug("Getting the default version from url {defaultVersionUrl}.", defaultVersionUrl);
 
             // get default version
-            var defaultVersionContent = httpClient
-                .GetStringAsync($"{defaultVersionUrl}{this.commonOptions.OryxSdkStorageAccountAccessToken}")
-                .Result;
+            string defaultVersionContent;
+            try
+            {
+                defaultVersionContent = httpClient
+                    .GetStringAsync($"{defaultVersionUrl}{this.commonOptions.OryxSdkStorageAccountAccessToken}")
+                    .Result;
+            }
+
+            // AggregateException is thrown by .Result
+            catch (AggregateException ae)
+            {
+                throw new AggregateException(
+                        $"Http request to '{defaultVersionUrl}' failed. {Constants.NetworkConfigurationHelpText}\n{ae}");
+            }
+
+            if (string.IsNullOrEmpty(defaultVersionContent))
+            {
+                throw new InvalidOperationException($"Http request to '{defaultVersionContent}' cannot return an empty result.");
```

Comment:
I believe this one should be `defaultVersionUrl`, and not `defaultVersionContent`